perun-base/src/main/resources/perun-base.xml

```diff
@@ -89,6 +89,7 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 					<prop key="perun.sms.program"></prop>
 					<prop key="perun.readOnlyPerun">false</prop>
 					<prop key="perun.DBInitializatorEnabled">false</prop>
+					<prop key="perun.userExtSources.persistent">PERUN,[\w\d]*</prop>
```

Comment:
It's used always as a backup, when no other config file is provided to Perun (``/etc/perun/perun.properties``) or such property is missing in it. So it's used by default for in-memory tests.